test/cgroups.bats

```diff
@@ -40,7 +40,7 @@ function teardown() {
 }
 
 @test "conmon custom cgroup" {
-	if [[ "$TRAVIS" == "true" ]]; then
+	if [[ "$CI" == "true" ]]; then
```

Comment:
Does `$CI` have the same semantics (i.e. set to "true") as `$TRAVIS`?

Reply:
Yes, that's completely the same. :+1: